bundles/org.openhab.binding.linky/src/main/resources/OH-INF/i18n/linky.properties

```diff
@@ -2,6 +2,8 @@
 
 binding.linky.name = Linky Binding
 binding.linky.description = Retrieves your energy consumption data from Enedis website
+binding.config.linky.requestBufferSize.label = Request Buffer Size
+binding.config.linky.requestBufferSize.description = Size of the buffer used to write requests (in bytes)
```

Comment:
Maybe you could make description here and in the XML file in sync.

Reply:
Done